pkg/domain/entities/engine_image.go

```diff
@@ -23,4 +23,7 @@ type ImageEngine interface {
 	Tag(ctx context.Context, nameOrId string, tags []string, options ImageTagOptions) error
 	Untag(ctx context.Context, nameOrId string, tags []string, options ImageUntagOptions) error
 	Search(ctx context.Context, term string, opts ImageSearchOptions) ([]ImageSearchReport, error)
+	ManifestCreate(ctx context.Context, names, images []string, opts ManifestCreateOptions) error
+	ManifestInspect(ctx context.Context, name string) error
+	ManifestAdd(ctx context.Context, opts ManifestAddOptions) error
```

Comment:
```suggestion
	ManifestCreate(ctx context.Context, names, images []string, opts ManifestCreateOptions) (<report>, error)
	ManifestInspect(ctx context.Context, name string)  (<report>, error)
	ManifestAdd(ctx context.Context, opts ManifestAddOptions)  (<report>, error)
```
You cannot present data to user in these functions, needs to be done in the manifest/*.go files.